apps/projects/app/components/Panel/NewIssue/NewIssue.js

```diff
@@ -100,17 +100,11 @@ class NewIssue extends React.PureComponent {
 
     const id = selectedProject > 0 ? reposIds[selectedProject - 1] : ''
 
-    const reGet = [
-      {
-        query: GET_ISSUES,
-        variables: { reposIds },
-      },
-    ]
+    // TODO: refetch Issues list after mutation
```

Comment:
I wonder if this solution is better than the former `reget` for the time being? Would you like to put some more context on this change to help understand please?

Reply:
The `reget` stuff wasn't working anymore, ever since we stopped using `GET_ISSUES`. 

I think actually making this work may be somewhat complicated; do we have an existing ticket for fixing this?

Reply:
I checked and I confirm that it stopped to work prior to this issue, as you pointed out, so I created [a new ticket](https://github.com/AutarkLabs/open-enterprise/issues/1694) to track it.

I think the TODO note is also a good idea to have as a reminder.